openPr.js

```diff
@@ -35,6 +35,7 @@ const getPRBody = (template, { newVersion, draftRelease, inputs }) => {
     tagsToUpdate: tagsToBeUpdated.join(', '),
     npmPublish: !!inputs['npm-token'],
     syncTags: /true/i.test(inputs['sync-semver-tags']),
+    author: inputs['actor-name'],
```

Comment:
Could we read the author from the GH context instead?
This requires to the user to set this info if I'm not wrong

Reply:
The user doesn't need to set this info as we use this default in action.yml

```yaml
  actor-name:
    description: "The name you want to see in the new release commit."
    required: false
    default: "${{ github.actor }}"
```

But yes, I too initially though of using the value from the `context` as that would be cleaner approach.

Reply:
I guess we initially added this `actor-name` input without much thought. But I think it wouldn't be so useful to let user modify the actor name so perhaps we could get rid of this input and use `github.actor` everywhere? /cc @simoneb

Reply:
I have updated the PR to use the `actor-name` from `context`